### ISSUE TITLE
Segmentation fault (core dumped) happened when I use more than one session.

### ISSUE THREAD

**Issue by calee88** [keras-team/keras#10693]
Hello. I'm new to Keras. I'm using TensorFlow as a backend. I'd like to load many models and use those in different threads. For different threads, I have to use different graphs, so I have to use different sessions. (Am I right?)

Here is a trimmed version of my code that can reproduce this error. What am I doing wrong?

I am using Keras-2.2.0 and tensorflow 1.8.0 without GPU on Ubuntu 16.04.4.

    import tensorflow as tf
    import numpy as np

    from keras.models import Sequential
    from keras.layers import Masking

    def predict():
        model = Sequential()
        layer = Masking(batch_input_shape=[None, 1])
        model.add(layer)

        x = np.array([[0]], dtype='int32')

        model.predict(x, batch_size=1)

    if __name__ == '__main__':
        with tf.Session():
            predict()

        with tf.Session():
            predict()

**Comment by calee88** [comment]
The error disappeared using tensorflow 1.9.0.

**Comment by LawrenceXia2008** [comment]
> The error disappeared using tensorflow 1.9.0.

this solution save my life i ve been puzzled without any idea for so long! thank you so much!
